apps/meteor/ee/app/livechat-enterprise/server/lib/AutoTransferChatScheduler.ts

```diff
@@ -76,35 +77,39 @@ class AutoTransferChatSchedulerClass {
 		const timeoutDuration = settings.get<number>('Livechat_auto_transfer_chat_timeout').toString();
 
 		if (!RoutingManager.getConfig().autoAssignAgent) {
-			return Livechat.returnRoomAsInquiry(room._id, departmentId, {
+			await Livechat.returnRoomAsInquiry(room._id, departmentId, {
 				scope: 'autoTransferUnansweredChatsToQueue',
 				comment: timeoutDuration,
 				transferredBy: await this.getSchedulerUser(),
 			});
+			return;
 		}
 
 		const agent = await RoutingManager.getNextAgent(departmentId, ignoreAgentId);
-		if (agent) {
-			return forwardRoomToAgent(room, {
-				userId: agent.agentId,
-				transferredBy: await this.getSchedulerUser(),
-				transferredTo: agent,
-				scope: 'autoTransferUnansweredChatsToAgent',
-				comment: timeoutDuration,
-			});
+		if (!agent) {
+			logger.error(`No agent found to transfer room ${room._id} which hasn't been answered in ${timeoutDuration} seconds`);
```

Comment:
Use a translatable message (not sure if that word exists lol)

Reply:
That's an error log for an async job, so it doesn't need translation (atleast for now until we support translation at log level!)